Sources/plutil/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+add_executable(plutil
```

Comment:
Missing file header here.

Reply:
Fixed in 97442722cc9aa16a7d6961a86fc2d6e26a3493ad